ksm.go

```diff
@@ -35,6 +33,9 @@ type ksmSetting struct {
 
 	// scanIntervalMS is the KSM scan interval in milliseconds.
 	scanIntervalMS uint32
+
+	// run describes if we want KSM to be on or off.
+	run bool
```

Comment:
Why put `run bool` down? Does this also conflict with golint rule?

Reply:
I agree that (in this case) it doesn't actually make any overall difference on the size of this struct, but the linter identified it as an issue and it's harmless to move it fwics.